src/common/src/util/addr.rs

```diff
@@ -41,6 +42,14 @@ impl From<SocketAddr> for HostAddr {
     }
 }
 
+impl From<MetaLeaderInfo> for HostAddr {
```

Comment:
It's a little bit weird to implement a `From` trait for `HostAddr` from `MetaLeaderInfo`.

Reply:
Alright. I implemented the logic in `risingwave_common::util::addr::leader_info_to_host_addr` instead